api/src/routes/settings.test.ts

```diff
@@ -157,6 +176,169 @@ describe('settingRoutes', () => {
       });
     });
 
+    describe('/update-my-email', () => {
+      beforeEach(async () => {
+        await fastifyTestInstance.prisma.user.updateMany({
+          where: { email: developerUserEmail },
+          data: {
+            newEmail: null,
+            emailVerified: true,
+            emailVerifyTTL: null,
+            emailAuthLinkTTL: null
+          }
+        });
+      });
+      test('PUT returns 200 status code with "success" message', async () => {
+        const response = await superRequest('/update-my-email', {
+          method: 'PUT',
+          setCookies
+        }).send({ email: 'foo@foo.com' });
+
+        expect(response?.body).toEqual({
+          message: 'flash.email-valid',
+          type: 'success'
+        });
+        expect(response?.statusCode).toEqual(200);
+      });
+
+      test("PUT updates the user's record in preparation for receiving auth email", async () => {
+        expect.assertions(6);
```

Comment:
Since none of the `expect` statements are inside an asynchronous function, do we need this? Same goes for line 283.

Reply:
It probably doesn't hurt. The test itself is async, and sometimes that does weird stuff.

Reply:
Yeah, it doesn't really hurt, but Niraj is right - we don't actually need them.

In this case I can't think of anything weird that could happen, so I'll get rid of them.